#goal
int32 save_files
---
#result
int32 result
---
#feedback
int32 number_of_faces